### PR TITLE
migrating meerkat status script from usersnfs/tiyani to utility

### PR DESCRIPTION
**Pull request by SirTiyani**
@ajoubertza and @tumelomangena please, review. The script is used by both operators and astronomers to get an overview of the meerkat receptors.

### CODE REVIEW

**Comment by ajoubertza** [comment]
@lanceWilliams, please can you review (instead of me)?

**Comment by samueltwum1** [comment]
Just a comment on the syntax: Consider running black to format your code to CAM recommended line length and recommended standards: `black --line-length 90`

**Comment by SirTiyani** [comment]
Thanks, @samueltwum1. I have applied the "black text formatting" as you suggested.

File: utility/meerkat_status.py [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+
+def sync_hrs_remaining(ants, band):
+    """Returns the hours left for which the digitiser remains synchronised"""
+    sensor = "dig_{}_band_time_remaining".format(band)
+    for ant in ants:
+        try:
+            time_left = ant.sensor[sensor].get_value()
+            if time_left:
+                break
+        except:
```
Comment (lanceWilliams) [inline utility/meerkat_status.py:19]: Use "except Exception" instead. Linters will typically complain if you just have an "except:" by itself.

File: utility/meerkat_status.py [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+            return sub_details
+
+
+def get_sensors(proxy, sensors):
+    """Returns sensor-value pairs for a given proxy and sensor list"""
+    sensor_values = {}
+    for sensor in sensors:
+        try:
+            sensor_values[sensor] = proxy.sensor[sensor].get_value()
+        except:
```
Comment (lanceWilliams) [inline utility/meerkat_status.py:69]: Use "except Exception" instead.

**Review by lanceWilliams — APPROVED** [review]
The rest looks ok

**Pull request merged**